opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -337,35 +337,42 @@ impl BatchLogProcessor {
                     match message_receiver.recv_timeout(remaining_time) {
                         Ok(BatchMessage::ExportLog(log)) => {
                             logs.push(log);
-                            if logs.len() == config.max_export_batch_size
-                                || last_export_time.elapsed() >= config.scheduled_delay
-                            {
+                            if logs.len() == config.max_export_batch_size {
+                                otel_debug!(
+                                    name: "BatchLogProcessor.ExportingDueToBatchSize",
+                                );
                                 let _ = export_with_timeout_sync(
-                                    remaining_time,
```

Comment:
passing export_timeout instead of remaining time. (Though the export method ignores this today, this is the right thing to be passed on).